sql_generators/glean_usage/templates/baseline_clients_yearly_v1.metadata.yaml

```diff
@@ -0,0 +1,23 @@
+{{ header_yaml }}
+friendly_name: Baseline Clients Yearly
+description: |-
+  Captures activity history of each client in 1-Year
+  windows for each submission date based on baseline pings.
+
+  Clustering fields: `normalized_channel`, `sample_id`
+
+  See also: `baseline_clients_daily`
+owners:
+  - frank@mozilla.com
+bigquery:
+  time_partitioning:
+    type: day
+    field: submission_date
+    require_partition_filter: true
+  clustering:
+    fields:
+    - normalized_channel
+    - sample_id
+schema:
+  derived_from:
+  - table: ['moz-fx-data-shared-prod', '{{ derived_dataset }}', 'baseline_clients_daily_v1']
```

Comment:
I haven't seen this before -- this forces the schema for this table to match `baseline_clients_daily_v1`?

Reply:
It does, but I kind of don't like it... I may remove it.